src/Controls/src/Core/HandlerImpl/Frame.Impl.cs

```diff
@@ -31,39 +31,5 @@ IShadow IView.Shadow
 				return null;
 			}
 		}
-
-		protected override Size MeasureOverride(double widthConstraint, double heightConstraint)
```

Comment:
This is copied from https://github.com/dotnet/maui/pull/6794

Without this change the code crashes